rbac/server/api/tasks.py

```diff
@@ -20,7 +20,7 @@
 
 from rbac.common import rbac
 
-from rbac.server.api.errors import ApiNotImplemented
+# from rbac.server.api.errors import ApiNotImplemented
```

Comment:
Same in this file.  Please remove all commented out endpoints from ApiNotImplemented